src/main/java/sirius/db/jdbc/UpdateStatement.java

```diff
@@ -163,26 +181,41 @@ public UpdateStatement setToToday(Mapping field) {
     }
 
     /**
-     * Adds an equals constraint to the query.
-     * <p>
-     * This will essentially generate a condition like {@code field = value}.
+     * Adds the given constraint to the query.
      *
      * @param field the field to check
+     * @param op    the operator to use
      * @param value the value to enforce
      * @return the query itself for fluent method calls
      */
-    public UpdateStatement where(Mapping field, Object value) {
+    public UpdateStatement where(Mapping field, Operator op, Object value) {
         prepareWhere();
         append(determineEffectiveColumnName(field));
-        if (value == null) {
+        if (value == null && op == Operator.EQ) {
```

Comment:
Not Equal condition should also be evaluated as "IS NOT NULL" (from my knowledge of Oracle DBs)

... else if (value == null && op == Operator.NE) {
   append(" IS NOT NULL");